board/safety/safety_hyundai_canfd.h

```diff
@@ -369,14 +380,24 @@ static const addr_checks* hyundai_canfd_init(uint16_t param) {
   }
 
   if (hyundai_longitudinal) {
-    hyundai_canfd_rx_checks = (addr_checks){hyundai_canfd_long_addr_checks, HYUNDAI_CANFD_LONG_ADDR_CHECK_LEN};
+    if (hyundai_canfd_hda2) {
+      hyundai_canfd_rx_checks = SET_ADDR_CHECKS(hyundai_canfd_hda2_long_addr_checks);
+    } else {
+      hyundai_canfd_rx_checks = hyundai_canfd_alt_buttons
+          ? SET_ADDR_CHECKS(hyundai_canfd_long_alt_buttons_addr_checks)
+          : SET_ADDR_CHECKS(hyundai_canfd_long_addr_checks);
```

Comment:
suspect it would read better like this instead of spread over a few lines
```suggestion
      hyundai_canfd_rx_checks = SET_ADDR_CHECKS(hyundai_canfd_alt_buttons ? hyundai_canfd_long_alt_buttons_addr_checks : hyundai_canfd_long_addr_checks);
```